bridges/ArtStationBridge.php

```diff
@@ -0,0 +1,84 @@
+<?php
+class ArtStationBridge extends BridgeAbstract {
+	const NAME = 'ArtStation';
+	const URI = 'https://www.artstation.com';
+	const DESCRIPTION = 'Fetches the latest ten artworks from a search query on ArtStation.';
+	const MAINTAINER = 'thefranke';
+	const CACHE_TIMEOUT = 3600; // 1h
+
+	const PARAMETERS = array(
+		'Search Query' => array(
+			'q' => array(
+				'name' => 'Search term',
+				'required' => true
+			)
+		)
+	);
+
+	public function getIcon() {
+		return 'https://www.artstation.com/assets/favicon-58653022bc38c1905ac7aa1b10bffa6b.ico';
+	}
+
+	public function getName() {
+		return self::NAME . ': ' . $this->getInput('q');
+	}
+
+	public function fetchSearch($searchQuery) {
+		$data = '{"query":"' . $searchQuery . '","page":1,"per_page":50,"sorting":"date",';
+		$data .= '"pro_first":"1","filters":[],"additional_fields":[]}';
+
+		$header = array(
+			'Content-Type: application/json',
+			'Accept: application/json'
+		);
+
+		$opts = array(
+			CURLOPT_POST => true,
+			CURLOPT_POSTFIELDS => $data,
+			CURLOPT_RETURNTRANSFER => true
+		);
+
+		$jsonSearchURL = self::URI . '/api/v2/search/projects.json';
+		$jsonSearchStr = getContents($jsonSearchURL, $header, $opts) or die('Could not fetch JSON for search query.');
```

Comment:
```suggestion
		$jsonSearchStr = getContents($jsonSearchURL, $header, $opts)
			or returnServerError('Could not fetch JSON for search query.');
```

Read more about [`returnServerError`](https://github.com/RSS-Bridge/rss-bridge/wiki/The-returnServerError-function).